apps-we-use/workable.md

```diff
@@ -0,0 +1,75 @@
+# Using Workable
+
+[Workable](https://www.workable.com/) is a all-in-one recruiting SaaS. We use it to source and evaluate candidates, track applicants and keep all information in one place so we can then make informed decisions on who to hire.
+
+## Hiring process
+
+### Overview
+
+We strive to have a bureaucracy- and ego- free hiring process that provides a welcoming experience for new Niteans. It's impossible to thoroughly test someone's fit with interviews and phone calls, so we rather do a couple of basic tests and then take them on for a [trial period].
+
+That should be your mindset when conducting interviews: you are not searching for an ideal candidate. You are on the lookout for big "no-no"'s. If someone's lacking some skills, they can be learned. If someone is used to different type of tools or processes, they can be changed. Look for problem solving prowess, ability to communicate what went wrong and ownership of mistakes.
+
+### Hiring workflow
+
+Workable guides us through the following stages of hiring a new Nitean:
+
+1. Sourcing
```

Comment:
This doesn't translate in Markdown well:

<img width="269" alt="screen shot 2018-05-14 at 09 33 24" src="https://user-images.githubusercontent.com/1309671/39983981-e2306970-5759-11e8-91f0-137a5cb6287e.png">